Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -151,6 +168,14 @@ namespace Multiplayer
         ;
     }
 
+    NetBindComponent::~NetBindComponent()
+    {
+        if (m_needsToBeStopped)
+        {
+            AZ_Warning("NetBindComponent", false, "NetBindComponent destructed without being stopped. Ensure MarkForRemoval is called before destroying a networked entity or disconnect is called before exiting.")                       
```

Comment:
Fyi, we have a cvar to gate this, feel to change the default value of it but it should apply here.
bg_AssertNetBindOnDeactivationWithoutMarkForRemoval

Reply:
Oh then we don't really need this at all. I'll remove it.